DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/Extensions/EditorExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Autodesk.AutoCAD.DatabaseServices;
+using Autodesk.AutoCAD.EditorInput;
+using Autodesk.AutoCAD.Geometry;
+
+namespace Speckle.Connectors.Autocad.HostApp.Extensions;
+
+public static class EditorExtensions
+{
+  public static void Zoom(this Editor editor, Extents3d ext)
+  {
+    if (editor == null)
+    {
+      throw new ArgumentNullException(nameof(editor));
+    }
+    using ViewTableRecord view = editor.GetCurrentView();
```

Comment:
Can we get a NL after the if statement

Reply:
Also, wat is going on with this using statement? isn't view being disposed before we start using it?

Reply:
nope, c#8 doesn't require braces, view is disposed at the end of the scope in which it is declared here

Reply:
oh I think this one is clearer with braces, much like single if statement body